openstack/image/v2/images/testing/fixtures_test.go

```diff
@@ -267,10 +267,11 @@ func HandleImageGetSuccessfully(t *testing.T) {
 			"disk_format": "qcow2",
 			"updated_at": "2014-05-05T17:15:11Z",
 			"visibility": "public",
-			"os_hidden": false,
+			"os_hidden": "False",
```

Comment:
This could do with a note about what we're simulating here also...

Reply:
done